.github/workflows/check-pytorch-package-indices.yml

```
@@ -7,6 +7,9 @@ on:
       - .github/workflows/check-available-pytorch-dists.yml
       - scripts/check_pytorch_package_indices.py
 
+  # Allows running this workflow manually from the Actions tab
+  workflow_dispatch:
```

Just my suggestion given that this will soon no longer my responsibility:

This comment is kinda unnecessary. It is correct, but having this comment there begs the question what is special about this one.

```suggestion
  workflow_dispatch:
```

I'm not going to add this everywhere, but the same comment pops up multiple times.

---

The comment was added primarily because I was copying the syntax from https://github.com/Slicer/Slicer/blob/1af67ed6f1411d9dac9f8d7fe246469bb0556569/.github/workflows/ci.yml#L11. It was only serving as reference to developers reading the code that may not know what `workflow_dispatch` as the name itself isn't super clear. Per your comment I have removed it from all the places where I added. Instead I have added information to the commit message body of the `Support manually triggering workflow` commit in this PR so that developers can git blame this line if they are ever curious and get back to the original commit which explains what it does and why it is helpful. If included a link to the GitHub actions documentation in that commit message body as well.